responsibleai/responsibleai/rai_insights/rai_insights.py

```diff
@@ -524,14 +531,37 @@ def _save_metadata(self, path):
         """
         top_dir = Path(path)
         classes = convert_to_list(self._classes)
+        feature_ranges = self._get_feature_ranges()
         meta = {
             _TARGET_COLUMN: self.target_column,
             _TASK_TYPE: self.task_type,
             _CATEGORICAL_FEATURES: self.categorical_features,
-            _CLASSES: classes
+            _CLASSES: classes,
+            _COLUMNS: list(self.test.columns),
+            _FEATURERANGES: feature_ranges
+
         }
         with open(top_dir / _META_JSON, 'w') as file:
             json.dump(meta, file)
+    
+    def _get_feature_ranges(self):
```

Comment:
Can you add unit tests for this change?

Reply:
+1.  Also, can you please add doc strings to the method?  We don't have them on all private methods, although we really should, and I think it's better to have more than fewer docs in the code.

Reply:
Thanks, added test and doc string.